### PR TITLE
Patch

### LINKED ISSUE #332: docker created from fairing uses python2 even base docker image has python 3 as default
/kind bug

**What steps did you take and what happened:**
I am using Trainjob API to deploy and run it remotely on GKE cluster. 
Following is the flow:
1. After running Trainjob API it is first creating docker (working correctly)
2. Then copying all the necessary dependencies and packaging it with docker (I am still not sure about this step)
3.Then pushing docker to specified registry (working correctly)
4. Then using the same docker to run it on GKE cluster (from here the problem starts)
 In step no 4 folling is the problem
````
Traceback (most recent call last):
  File "/app/function_shim.py", line 4, in <module>
    from enum import Enum
ImportError: No module named enum
```   
I looked up the problem & following is conclusion:
1. enum package generally gives problem in python2
2. So I made sure my base docker image will use python3 as default. So should the docker generated step 2, but I think this is not the case. 

**What did you expect to happen:**
Function inside the docker should work just like the function working locally. 

**Anything else you would like to add:**
Docker generated in step 2 i.e. by Faring should use python3 by default 
I think so make change in Docker command python to python3.  (below log may help)
`Docker command: ['python', '/app/function_shim.py', '--serialized_fn_file', '/app/pickled_fn.p']`



**Environment:**

- Fairing version: 0.5.3
- Kubeflow version: 0.5

### PR DESCRIPTION
**Pull request by Akashdesarda**
**What this PR does / why we need it**:
make docker run python3 as default. I faced lot of issue due to python2 which got resolved when the docker ran on python3
**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #332

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note
None
```

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/fairing/333)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -15,4 +15,6 @@ again.
 All submissions, including submissions by project members, require review. We
 use GitHub pull requests for this purpose. Consult
 [GitHub Help](https://help.github.com/articles/about-pull-requests/) for more
-information on using pull requests.
\ No newline at end of file
+information on using pull requests.
+
+-[Akash Desarda](https://github.com/Akashdesarda)
```
Comment (jinchihe) [inline CONTRIBUTING.md]: Seems no need to add this?

**Comment by jinchihe** [comment]
@Akashdesarda I think you can update your env to install or alias python to python3? or create a python3 virtual environment to do? Personally I think that's OK because since the `python` means `python3` in some env. Thanks.

**Comment by Akashdesarda** [comment]
@jinchihe I already all things that you mentioned like using alias, python 3 venv, removing python 2 completely, still not successful

**Comment by jinchihe** [comment]
> @jinchihe I already all things that you mentioned like using alias, python 3 venv, removing python 2 completely, still not successful

What'a the problem then? thanks.

**Comment by karthikv2k** [comment]
Closing this pull request for now. Please discuss the problem and agree on the solution in #332 and then create a PR for it.

**Pull request closed without merging**